zerver/tests/test_message_topics.py

```diff
@@ -230,10 +231,14 @@ def test_topic_delete(self) -> None:
             },
         )
         self.assert_json_error(result, "Must be an organization administrator")
-        self.assertEqual(self.get_last_message().id, last_msg_id)
+        # Filter out "stream events" notifications
+        last_message = messages_excluding_stream_notifications().last()
```

Comment:
This works, but is a bit fragile.  I think a cleaner approach would be just to have saved the message ID for our moved message from `self.send_stream_message`, above, and then more explicitly verifying that the message was (not) deleted via, say, a `Message.objects.exists(id={message_id})` check, rather than kinda hackishly using `self.get_last_message().id` for this verification?